packages/mdc-radio/_mixins.scss

```diff
@@ -53,6 +53,9 @@ $mdc-radio-ripple-target: ".mdc-radio__ripple";
     @include mdc-radio-unchecked-stroke-color($mdc-radio-unchecked-color, $query);
     @include mdc-radio-checked-stroke-color($mdc-radio-baseline-theme-color, $query);
     @include mdc-radio-ink-color($mdc-radio-baseline-theme-color, $query);
+    @include mdc-radio-disabled-unchecked-stroke-color($mdc-radio-circle-color, $query);
```

Comment:
nit: `$query: $query` It is a good practice to name the optional parameters so it still works when we add a new parameter to this mixin.